slsim/lens.py

```diff
@@ -670,19 +698,25 @@ def source_light_model_lenstronomy(self, band=None):
             self._source_type == "extended"
             or self._source_type == "point_plus_extended"
         ):
-            if self.light_profile == "single_sersic":
-                source_models["source_light_model_list"] = ["SERSIC_ELLIPSE"]
+            source_models_list = []
+            kwargs_source_list = []
+            for source in self.source:
```

Comment:
I would re-structure this since we might want different source types for different sources supported

Reply:
There are more complex issues when we try to allow difference source type. So, I think it is good to work on this with a separate PR.